File: translation/flutter/label.dart

//#pragma: imports
import 'dart:convert';

//#pragma: execution
GestureDetector(onTap:(){ dynamic event;##TRIGGERS## }, child:Text("${StringWrapper(##SOURCE##).value != '' ? StringWrapper(##SOURCE##).value : '[[label]]'}", style: Theme.of(context).textTheme.headlineMedium)),

File: translation/react/label.js

//#pragma: declaration
let MyLabel = ({text, click}) => <div onClick={click}>{text}</div>
//#pragma: execution
<div><MyLabel text={datasource[[source]] ? datasource[[source]].getValue() +'' : '[[label]]'} click={() => {##TRIGGERS##}}> </MyLabel></div>